### ISSUE TITLE
Proposed update to PBS GDPR restriction processing

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#1748]
Xandr has suggested that there's a detail in the original Prebid Server GDPR enforcement that needs to be revisited.

The suggestion is summarized in the "Full Enforcement" truth table in the appendix of https://docs.google.com/document/d/1fBRaodKifv1pYsWY3ia-9K96VHUjd8kKvxZlOsozm8E/edit#

Basically, we think the [IAB's document](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md) is self-contradictory on this point:

On one hand it says:
```
RestrictionType 1 Require Consent (if Vendor has declared the Purpose IDs
legal basis as Legitimate Interest and flexible)
```
But then right next to that it says
```
If this value is 1 and vendor is registered under Legitimate Interest for that Purpose
then the vendor should not process for that Purpose.
```
The original PBS+GDPR spec implemented the intent of the first statement. The proposed revision focuses on the second statement, with the proposed changes in green:

<img width="955" alt="Screen Shot 2021-03-10 at 11 06 05 AM" src="https://user-images.githubusercontent.com/4412838/110672074-34c36000-819d-11eb-8982-1e6a79b6781f.png">
<img width="965" alt="Screen Shot 2021-03-10 at 11 06 11 AM" src="https://user-images.githubusercontent.com/4412838/110672120-47d63000-819d-11eb-9ba5-f784cfe7be36.png">

Community input welcome.

**Comment by bretg** [comment]
Confirmed with Prebid's privacy lawyers that this proposal is a good reading of the IAB's document.

I've updated Prebid's doc with the rule change here and will open a ticket for the change in PBS-Java

**Comment by SyntaxNode** [comment]
@bretg Is this complete in PBS-Java? Looks like it was already merged in.

**Comment by bretg** [comment]
Released with PBS-Java 1.61
